src/modules/commander/commander_params.c

```diff
@@ -313,18 +313,16 @@ PARAM_DEFINE_INT32(COM_ARM_SWISBTN, 0);
 /**
  * Battery failsafe mode
  *
- * Action the system takes on low battery. Defaults to off
+ * Action the system takes at critical battery.
```

Comment:
@julianoes What do you think about referencing to BAT_EMERGEN_THR?
I'm not sure if we generally should / do that but in this case I find "critical battery" quite meaningless.

Reply:
Agreed. Can you make a PR?

Reply:
@julianoes No....ok, ok :-)